scss/bitstyles/atoms/badge/Badge.stories.js

```diff
@@ -269,3 +269,20 @@ PositiveButtonSmall.args = {
   onClick: dummyOnClick,
   sizeVariant: ['small'],
 };
+
+const TemplateBadgeWithLabel = (args) => {
+  const fragment = document.createElement('div');
+  const mainContent = document.createElement('span');
+  mainContent.setAttribute('id', 'main');
+  mainContent.classList.add('u-margin-m-left');
+  mainContent.innerHTML = 'Trailing label';
```

Comment:
Something like `trailingLabel` could be a better name for this, wdyt? And if the `id` attribute isn’t needed for this component, let’s remove it (so we get clearer html in the code examples)